rest/rest.go

```diff
@@ -193,6 +186,22 @@ func RunServer(ctx context.Context, grpcPort int, httpPort int, corsOpts ...CORS
 	return srv.Serve(sock)
 }
 
+// StopServer is in charge of stopping the REST API.
+func StopServer(ctx context.Context) {
```

Comment:
Exported on purpose?

Reply:
Kind of. Since RunServer was exported, I also exported StopServer.

Reply:
Ok. Thus, engine could stop REST server manually. Sounds reasonable